erpnext/hr/doctype/payroll_entry/payroll_entry.py

```diff
@@ -66,6 +66,7 @@ def get_emp_list(self):
 				where
 					t1.name = t2.employee
 					and t2.docstatus = 1
+					and t1.status = "Active"
```

Comment:
what if the employee left in the middle of the month? you should check for the leaving date as well.

Reply:
in case the payroll entry is for the entire month & an employee has left on the 15th of the month. 

he should be fetched however, his salary is computed till 15th only.

Reply:
In that case, employees need to be active with relieving dates, until unless payroll is not processed.

Reply:
I have updated the PR please have a look

Reply:
@Anurag810 why would the status be active for an employee after he has left? this is a bug in the system and needs to be fixed.

Reply:
Is there a separate task for this bug, because I remember this being discussed?

Reply:
> Is there a separate task for this bug, because I remember this being discussed?

@neilLasrado this is the task https://bloomstack.com/desk#Form/Task/TASK-2021-00419

Reply:
I have left some comments on the task. I can't seem to replicate the issue.

I was able to process payroll for an Employee marked as left. @Anurag810

Reply:
@neilLasrado there is no separate task for this bug, it was raised as an issue. 
yeah right, employees should be marked as "Left" based on their relieving date. and the query filter will be on relieving dates too for employees.

reference: https://bloomstack.com/desk#Form/Issue/ISS-2021-00312

@MyuddinKhatri  @neilLasrado we need to separate/differentiate between tasks and issues. some time important fixes got stuck in the backlog.
> 
> @neilLasrado this is the task https://bloomstack.com/desk#Form/Task/TASK-2021-00419